pkg/contracts/src/RegistryCommunity.sol

```diff
@@ -154,25 +155,31 @@ contract RegistryCommunity is ReentrancyGuard, AccessControl {
         address _strategyTemplate;
     }
 
-    //TODO: can change to uint32 with optimized storage order
-    FAllo public allo;
-    IRegistry public registry;
-    IERC20 public gardenToken;
+    //DONE: grouping and order of variables
+    // Smaller uints and bools packed together
+    bool public isKickEnabled; // 1 byte, consider aligning with other small data types or bools if available
 
+    // 256-bit (32 bytes) uints and bytes packed together
     uint256 public registerStakeAmount;
     uint256 public communityFee;
-    address public feeReceiver;
     bytes32 public profileId;
-    bool public isKickEnabled;
+
+    // Address types packed (20 bytes each)
+    address public feeReceiver;
     address public registryFactory;
     address public strategyTemplate;
     uint256 public cloneNonce;
+    address payable public pendingCouncilSafe;
 
-    address payable public pendingCouncilSafe; //@todo write test for change owner in 2 step
+    // Contract interfaces (addresses internally, hence 20 bytes each)
+    IRegistry public registry;
+    IERC20 public gardenToken;
     ISafe public councilSafe;
+    FAllo public allo;
 
+    // Dynamic string types (variable size, stored last)
     string public communityName;
-    string public covenantIpfsHash; //@todo maybe should be bytes32
+    string public covenantIpfsHash;
```

Comment:
bytes32 will be more efficeint but will need change also subgraph

Reply:
so we add or not for now ? and keep comment

Reply:
lets keep